src/main/java/mif/vu/ikeea/Enums/ERestrictionDays.java

```diff
@@ -0,0 +1,13 @@
+package mif.vu.ikeea.Enums;
+
+public enum ERestrictionDays {
+    DEFAULT(3);
+
+    private final int value;
+
+    ERestrictionDays(final int newValue) {
+        value = newValue;
```

Comment:
Nelabai suprantu, kaip visa taip sudentingai daryti? Tau tereikia `ApplicationUser` 46 eilutej pakeisti defaulta ir tam integeriui reiksme priskirti defaultine, that's all